Assets/General/Scripts/Console/ConsoleController.cs

```diff
@@ -79,8 +93,77 @@ private void ToggleConsole()
         }
     }
 
+    #region FreelookCam
+    /// <summary>
+    /// The player unposseses the character
+    /// and become a free look cam that can
+    /// fly arround
+    /// </summary>
+    private void ToggleFreeLookCam()
+    {
+        if (!_isInDeveloperMode) return;
+
+        if (!_isInFreeLookCamMode)
+        {
+             _spawnedFreeLookCam = EnterFreeLookCam();
+            _toggleFreeLookCamButton.GetComponentInChildren<TMP_Text>().text = "Exit Free Look Cam";
+        }
+        else if (_isInFreeLookCamMode)
+        {
+            ExitFreeLookCam();
+            _toggleFreeLookCamButton.GetComponentInChildren<TMP_Text>().text = "Enter Free Look Cam";
+        }
+
+    }
+    /// <summary>
+    /// Puts the player in free look cam
+    /// mode
+    /// </summary>
+    private GameObject EnterFreeLookCam()
+    {
+        if (!_isInDeveloperMode) return null;
+
+        CinemachineVirtualCamera _playerVirtualCam = GameObject.FindObjectOfType<CinemachineVirtualCamera>();
+
+        //spawn free look cam
+        GameObject _tempFreeLookCam = Instantiate(_freeLookCam,
+            GameObject.FindGameObjectWithTag("MainCamera").transform.position, Quaternion.identity,
+            GameObject.FindObjectOfType<BoatScriptTag>().transform);
+
+        //stop actual player from moving
```

Comment:
What is this comment referring to? There is no line directly under this

Reply:
When the free cam moves, the player also moves. I wanted to stop that, but realized I couldn't sync it wasn't part of my task. and wasn't a big issue. I planned on talking to the others about it to get it as a task

Reply:
stopping the real player's body from moving. it wasn't a big issue, just a preference so I decided it could be done later since that wasn't part of the task.